uber/templates/preregistration/stripeForm.html

```diff
@@ -1,16 +1,161 @@
-<form class="stripe" method="post" action="{{ action }}">
-    <input type="hidden" name="payment_id" value="{{ payment_id }}" />
-    {{ csrf_token() }}
-    <script
-        src="https://checkout.stripe.com/v2/checkout.js" class="stripe-button"
-        data-key="{{ c.STRIPE_PUBLIC_KEY }}"
-        {% if email %} data-email="{{ email }}"{% endif %}
-        {% if c.DEV_BOX %} data-bitcoin="true" {% endif %}
-        data-zip-code="true"
-        data-allow-remember-me="false"
-        data-amount="{{ charge.amount }}"
-        data-name="{{ c.EVENT_NAME_AND_YEAR }} {{ regtext }}"
-        data-description="{{ charge.description }}"
-        data-image="../static/theme/stripe-logo.png">
-    </script>
-</form>
+
+<style>
+.StripeElement {
+  box-sizing: border-box;
+
+  height: 40px;
+
+  padding: 10px 12px;
+
+  border: 1px solid transparent;
+  border-radius: 4px;
+  background-color: white;
+
+  box-shadow: 0 1px 3px 0 #e6ebf1;
+  -webkit-transition: box-shadow 150ms ease;
+  transition: box-shadow 150ms ease;
+}
+
+.StripeElement--focus {
+  box-shadow: 0 1px 3px 0 #cfd7df;
+}
+
+.StripeElement--invalid {
+  border-color: #fa755a;
+}
+
+.StripeElement--webkit-autofill {
+  background-color: #fefde5 !important;
+}
+</style>
+<button class="btn btn-info" onClick="callStripeAction();">
+<span class="display: block; min-height: 30px;">Pay with Card</span>
+</button>
+<div id="stripeModal" class="modal" style="color:black;" tabindex="-1" role="dialog" data-backdrop="static">
```

Comment:
`data-backgroup="static"` prevents the modal from being closed when you click outside it. I figured people are more likely to accidentally click outside the modal than to want to cancel the modal. You can still close the modal with the "Cancel" button or by pressing Esc.